app/controllers/alchemy/pages_controller.rb

```diff
@@ -1,30 +1,63 @@
 module Alchemy
   class PagesController < Alchemy::BaseController
+
+    # Redirects to signup path, if no admin user is present yet
+    before_action if: :signup_required? do
+      redirect_to Alchemy.signup_path
+    end
+
     before_action :enforce_primary_host_for_site
+
+    before_action :enforce_no_locale, only: [:index, :show],
+      if: :default_locale_requested?
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.